pingpong/test_models_threads.py

```
@@ -331,12 +336,19 @@ async def test_get_thread_by_class_id_preloads_export_user_fields(db):
+    assert "anonymous_link_id" not in unloaded
+    assert "anonymous_link" not in unloaded
     assert "display_name" not in unloaded
     assert "first_name" not in unloaded
     assert "last_name" not in unloaded
     assert "email" not in unloaded
     assert loaded_user.display_name == "Export User"
     assert loaded_user.email == "export-user@example.com"
+    assert loaded_user.anonymous_link.name == "Research cohort"
+    assert (
+        loaded_user.anonymous_link.share_token
+        == "019afc9f-7634-7621-8aca-50c93f6dd956"
```

Ruff (0.15.13)

[error] Possible hardcoded password assigned to: "share_token"

(S105)